client/assets/styles/_sidebar.scss

```
@@ -47,73 +45,66 @@
     transition-timing-function: ease;
     transition-duration: 1s;
 }
- 
- 
+
+
 .logo {
    height: 72px !important;
    padding: 18px 10px 18px 10px !important;
 }
 
- 
 .title.item {
     padding: 18px 18px 18px 18px !important;
     transition: 1.5s;
     &.active {
         border-left: 9px solid #40aaca;
         box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24) !important;
         transition: all 0.3s cubic-bezier(.25,.8,.25,1) !important;
-    }        
+    }
 }
- 
- 
+
+
 .dropdown .menu .header {
    padding-top: 3.9px !important;
    padding-bottom: 3.9px !important;
 }
-
 .theme_primary {
     background-color: #1d5261 !important;
     .item {
         color: white !important;
-    }        
+    }
     .divider__title {
         font-weight: 500 !important;
         padding: 28px 12px 28px 19px !important;
         background-color: #2a6d81 !important;
         text-transform: uppercase !important;
         letter-spacing: 2.74px;
     }
-        
+
 }
 .brand__logo {
-    width: 40px !important;
-    height: 40px !important;
+    width: 38px !important;
+    height: 38px !important;
 }
-
 .brand__name {
-    height: 38px !important;
-    width: 218px !important;
-    font-size: 14px !important;
+    height: 35px !important;
+    width: 204px !important;
+    font-size: 13px !important;
     color: white !important;
     margin-left: 8px;
     margin-top: 6px;
     text-indent: 1px;
 }
-
 .sidebar.theme_primary {
-     width: 304px !important;
+     width: 283px !important;
 }
-
 .float-left {
     float: left;
 }
-
 .ui.accordion {
     .title {
         font-weight: 200 !important;
-    }        
+    }
 }
-
 .content.show {
```

```suggestion
.sidebar .content.show {
```
To help localize it to the sidebar as I discovered that is the only place it is needed.
Also, you'll need to change the corresponding `.content {` style on line 115 from
`.content {`
to
`.sidebar .content {`
This is important because these styles affect all .content. For example, the `div.content` in <Loader> from `semantic-ui-react` doesn't display because of this, the content of modals are equally hidden. You'll have to go add !important to override the styles that are affecting them.